packages/wonder-blocks-layout/components/spring.js

```diff
@@ -11,7 +11,7 @@ import {View} from "@khanacademy/wonder-blocks-core";
 
 export default class Spring extends React.Component<{}> {
     render() {
-        return <View style={styles.grow} />;
+        return <View aria-hidden="true" style={styles.grow} />;
```

Comment:
Huh... does having these be hidden have any impact if there is nothing inside of them and they don't receive focus?

Reply:
Not necessarily, but since we can't test every screen reader or assistive tech, it seems prudent to be as explicit as we can.

The biggest effect this has is perf. The browser will stop searching at nodes marked as hidden. If there were more empty children here, it would go through all of them. Marking as hidden short-circuits that effort.